js/fetch.js

```diff
@@ -11,15 +11,8 @@
  * @class
  */
 async function fetch(url, options = {}) {
-	// const nodeVersion = process.version.match(/^v(\d+)\.*/)[1];
-	// if (nodeVersion >= 18) {
-	// 	// node version >= 18
-	// 	return global.fetch(url, options);
```

Comment:
we cant switch to global.fetch now?

Reply:
I'd love to get rid of the dependency and my calendars also work with global.fetch, but I'm not sure if they all do.

@khassel, you had dealt with it more intensively. What do you think?

Reply:
I will prepare an extra PR for node-fetch removal. Then we can continue the discussion there. That would have the advantage that we could accept this PR here.

Reply:
see https://github.com/MichMich/MagicMirror/pull/2961 and https://github.com/MichMich/MagicMirror/pull/2952

We could try it again and see what happens with the e2e tests, they are meanwhile unstable anyway ...

Reply:
To continue the fetch topic I created PR #3172.